frontend/app/helpers/spree/orders_helper.rb

```diff
@@ -3,5 +3,12 @@ module OrdersHelper
     def order_just_completed?(order)
       flash[:order_completed] && order.present?
     end
+
+    def order_has_free_shipping?(order)
```

Comment:
@kaliszKrzysztof let's move this into the `Order` model please as `has_free_shipping?` method